src/app/core/admin/audit/audit.component.ts

```diff
@@ -0,0 +1,271 @@
+import { Component, OnInit } from '@angular/core';
+import { DatePipe } from '@angular/common';
+
+import _isString from 'lodash/isString';
+import { utc } from 'moment';
+import { BsModalRef, BsModalService } from 'ngx-bootstrap';
+import { Observable, forkJoin } from 'rxjs';
+
+import { AuditService } from './audit.service';
+import { AuditViewChangeModal, AuditViewDetailModal } from './audit-view-change.component';
+import { PagingOptions, SortDisplayOption, SortDirection, SortableTableHeader, TableSortOptions, PagingResults } from '../../../common/paging.module';
+import { AdminUsersService } from '../user-management/admin-users.service';
+import { AuditOption } from './audit.classes';
+import { AdminTopics } from '../admin-topic.model';
+
+
+@Component({
+	selector: 'audit',
+	styleUrls: ['audit.scss'],
+	templateUrl: './audit-list.component.html'
```

Comment:
Probably want this to be `audit.component.html` to clearly link this component and template.

Reply:
I moved and renamed all corresponding files to`list-audit-entries`